subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/versions/AndroidGradlePluginVersions.groovy

```diff
@@ -55,6 +55,10 @@ class AndroidGradlePluginVersions {
         }
     """
 
+    private static final VersionNumber AGP_7_0_VERSION_NUMBER = VersionNumber.parse('7.0.0')
+    private static final VersionNumber AGP_7_3_VERSION_NUMBER = VersionNumber.parse('7.3.0')
+    private static final VersionNumber KOTLIN_1_5_20_VERSION_NUMBER = VersionNumber.parse('1.5.20')
```

Comment:
Nit: I think we should drop `_VERSION_NUMBER` here. It feels like we are doing systems Hungarian notation.

Reply:
I dropped the `VERSION_NUMBER` suffix.